### DISCUSSION TITLE
CleanRayCommand should support versions of Laravel older than Laravel 10

### DISCUSSION THREAD

**Discussion by joshlewis** [spatie/laravel-ray#361] [Ideas]
CleanRayCommand relies on `Illuminate\Support\Facades\Process`, which was introduced with Laravel 10.

I think it would be really great if the code could support running the `ray:clean` command for people who are using a version of Laravel older than Laravel 10.

**Comment by freekmurze** [comment]
If it's very simple and tested, I'd consider merging it in
